### DISCUSSION TITLE
OneGraph Explorer Status?

### DISCUSSION THREAD

**Discussion by SachaG** [graphql/graphiql#3068] [General]
I'm a huge fan of the OneGraph Explorer. In fact it's the main way I interact with GraphiQL, and I've given up on every other GraphQL IDE out there simply because they don't have a similar feature. 

But with GraphiQL 2, I'm a bit uncertain about the status of the explorer? I know OneGraph doesn't exist anymore, so is the plugin still maintained? Are there any plans to make it an official part of the GraphiQL core? (which would make sense to me since it's the part of the IDE I use the most). And can I safely upgrade to GraphiQL 2 without losing the Explorer feature?

**Comment by LekoArts** [comment]
Do you mean https://github.com/graphql/graphiql/tree/main/packages/graphiql-plugin-explorer ?

**Reply by SachaG** [reply]
Yes that's what I mean. That page still links to the OneGraph repo, so I'm just wondering about the status of the project.

**Reply by LekoArts** [reply]
I guess it links to it because they were the original authors and the package is still used.

> so is the plugin still maintained

Probably not atm

> Are there any plans to make it an official part of the GraphiQL core? (which would make sense to me since it's the part of the IDE I use the most)

It's a plugin so probably not

> And can I safely upgrade to GraphiQL 2 without losing the Explorer feature

You need to install https://github.com/graphql/graphiql/tree/main/packages/graphiql-plugin-explorer and configure but other than that it'll work just as before.

You can see it in action inside Gatsby here: https://www.gatsbyjs.com/docs/how-to/querying-data/running-queries-with-graphiql/#example-video-of-using-graphiql

**Comment by acao** [comment]
@SachaG Hey there, great question!

I originally advocated for taking it over, and got the go-ahead to do so from onegraph in 2021

However my co-maintainers decided to take things in a different direction, and I was busy with work and wasn't as involved in the future of this as I'd like to be

I would still like to maintain it, but it wouldn't be part of the "graphiql core", or perhaps even a graphql org repository.

In fact, doc explorer, history, many features will all be moved to plugins soon

**Reply by SachaG** [reply]
Thanks for the details! In the meantime I was able to install it as a plug-in on the latest version of GraphiQL and it seems to work as before, so that's great.

**Reply by acao** [reply]
yes! I recently made the implementation much simpler, no need for passing hooks to the plugin, there was a pattern for a while that led to cursor jumps and other bugs
